db/Dockerfile.in

```diff
@@ -1,2 +1,3 @@
 FROM mysql:5.7
 ADD ${CONTENT_DB} /docker-entrypoint-initdb.d/
+ADD duplicate-db.sh /docker-entrypoint-initdb.d/
```

Comment:
As per hadolint, if you're not expecting Docker to perform remote pulls or to extract .gz during an ADD command, use COPY: https://github.com/hadolint/hadolint/wiki/DL3020
```
dev/stdin:3 DL3020 Use COPY instead of ADD for files and folders
```

This error should have been picked up if you have run `make init` before committing changes.